### PR TITLE
Documention for get categorymembers API

### PR DESCRIPTION
**Pull request by sra-013**
Hi Srishti,

I have added some code files. Please review them. 

Thanks,
Supriya.

### CODE REVIEW

File: get_category_items.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+"""
+    MediaWiki Action API Code Samples
+    Demo of `get_category_list` module : Sending get request to list pages that belong to a
```
Comment (srish) [inline get_category_items.py]: Change this statement to 

> Demo of `Categorymembers` module: List twenty items in a category.

File: get_category_items.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python3
+
+"""
```
Comment (srish) [inline get_category_items.py]: Add name of the file after the quotes like [here](https://github.com/srish/MediaWiki-Action-API-Code-Samples/blob/master/clientlogin.py). Same goes for other two files too.

File: get_recent_category_items.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+
+"""
+    MediaWiki Action API Code Samples
+    Demo of `get_category_list` module : Sending get request to list pages that belong to a
```
Comment (srish) [inline get_recent_category_items.py]: Change this statement to

> Demo of `Categorymembers` module: Get ten articles most recently added to a category. And, same comment goes for the `get_subcategories.py` file as well.

File: get_category_items.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    'list' : "categorymembers",
+    'cmtitle' : "Category:Physics",
+    'cmlimit' : 20,
+    'format':"json"
+}
+
+R = S.get(url=URL, params=PARAMS)
+DATA = R.json()
+
+print DATA
```
[the remark this answers is not in the document]
Comment (sra-013) [reply get_category_items.py]: Oh right, I still use Python 2 on my system so Pylint asked me to format the code accordingly. I'll update.

**Comment by srish** [comment]
@sra-013 Looks neat! Thanks! A few minor things to nit pick on:
* You probably forgot to run the code via `Pylint` after making changes and that is why `Pylint` reported trailing whitespaces in some places 
* You created four different commits with message "formatting..". As they were all minor changes you could commit them all at once . Also, this might be insightful https://chris.beams.io/posts/git-commit/ :-)

**Pull request merged**